torchaudio/models/wav2vec2/model.py

```diff
@@ -20,15 +20,20 @@ class Wav2Vec2Model(Module):
         encoder (torch.nn.Module):
             Encoder that converts the audio features into the sequence of probability
             distribution (in negative log-likelihood) over labels.
+
+        aux (Optional[torch.nn.Module]):
```

Comment:
nit: should we follow the scheme described at #1746? e.g.

```suggestion
        aux (torch.nn.Module or None, optional):
```

Reply:
good catch.